k8s: add levels

hunchback opened the pull request:
![image](https://user-images.githubusercontent.com/2760739/69706562-a68aae00-1100-11ea-81e3-a741ce6a0a35.png)

eranra commented on src/Config.ts:
```diff
@@ -1,140 +1,174 @@
+const LEVEL_K8S_FEDERATION = 100
+const LEVEL_K8S_CLUSTER = 101
+const LEVEL_K8S_NODE = 102
+const LEVEL_K8S_POD = 103
```
Split into two views:

1. k8s deployment view
2. k8s logical view

in (1) we need at least another layer for deployments and not show things that are logical
in (2) we need at least another layer for services and ingress to show end-points

It is ok not to show all objects in all views. so for example I would advice to move the storage out to maybe another view that is based on storage only

hunchback replied:
there were many view in the previous ui:

- storage
- network
- compute
- deployment
- service

this will be addressed in a separate PR

hunchback commented:
> * Is there a way to create a view that show pod --> container --> docker or run/c --> host ... this will be very useful

I agree - if we want to connect k8s to L2 we need such a view - of course not in the scope of this PR

eranra commented:
Do we have anything we can connect k8s to infra. or we need to change backend as base ?

safchain commented:
Just need to rebase

hunchback commented:
I have rebased and updated a little ... all working as expected:

![image](https://user-images.githubusercontent.com/2760739/71323021-3bea4980-24d7-11ea-9973-65b236bde21f.png)
